test/support/generator/random-value-generator.ts

```diff
@@ -20,10 +25,10 @@ export class RandomValueGenerator {
     return this.chance.url({ protocol: 'https' });
   }
 
-  stringOfNumericCharacters(minLength?: number): string {
+  stringOfNumericCharacters(minLength?: number, maxLength?: number): string {
```

Comment:
Can we change the function signature to

```javascript
stringOfNumericCharacters({ minLength, maxLength }: { minLength?: number; maxLength?: number}: string
```
please?

That way the caller has to call this function with named parameters so no-one needs to memorise if max or min comes first